File: Solutions/140035/java/program.java
// Copy from Quera

import java.util.Scanner;

public class Main {

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int a = scanner.nextInt() , b = scanner.nextInt() , c = scanner.nextInt();
        if ((a%2 == 0) || ((b % 2 == 0) && (c % 2 == 0))){
            System.out.print("good");}
        else {
            System.out.print("bad");
        }
        

    }
}

File: Solutions/140035/c++/program.cpp
// Copy from Quera

#include <iostream>
#include <iomanip>
using namespace std;
int main()
{
    int A, B, C;
    cin >> A >> B >> C;

    if (A % 2 == 0 || (B % 2 == 0 && C % 2 == 0))
    {
        cout << "good";
    }
    else
    {
        cout << "bad";
    }
    return 0;
}